core/src/Microsoft.Identity.Core.csproj

```diff
@@ -123,4 +124,7 @@
       <Version>27.0.2</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Realm\" />
+  </ItemGroup>
```

Comment:
why is this needed?

Reply:
Rest assured that I was not the one who manually created this. It was somehow automatically generated by my VS environment and I wasn't able to give it much thought. Now looking at it, it is probably this [known issue](https://github.com/dotnet/sdk/issues/1105) with this [root cause](https://github.com/dotnet/project-system/issues/1949#issuecomment-381939811). I'll revert this part of change in upcoming commit. #ConsiderItDone